src/main/java/racing/Car.java

```diff
@@ -0,0 +1,40 @@
+package racing;
+
+public class Car {
+
+    private int position;
+    private final static int MINIMUM_CONDITIONS = 4;
+    private String carName;
+
+    public Car(String inputCarName) {
+        this(inputCarName, 0);
+    }
+
+    public Car(String inputCarName, int position) {
+        this.carName = inputCarName;
+        this.position = position;
+    }
+
+    private void decidePosition(boolean greaterCheck) {
+        if (greaterCheck) {
+            this.position += 1;
+        }
+    }
+
+    public int getPosition() {
+        return this.position;
+    }
+
+    public String getCarName() {
+        return this.carName;
+    }
+
+    public void move(int number) {
+        decidePosition(number >= MINIMUM_CONDITIONS ? true : false);
+    }
```

Comment:
단순히 인라인을 해버려 오히려 코드가 무엇을 하려고 하는지 알아보기가 힘들어 졌어요.
아래 코드 처럼 변경해 보는건 어떨까요?
```suggestion
     public void move(int number) {
        if (canMove(number)) {
            this.position++;
        }
    }

    private boolean canMove(int number) {
        return number >= MINIMUM_CONDITIONS;
    }
```

Reply:
완료